knexfile.js

```diff
@@ -0,0 +1,19 @@
+
+module.exports = {
+  development: {
+    client: 'pg',
+    connection: 'postgres://localhost/palettepicker',
+    migrations: {
+      directory: './migrations'
+    },
+    seeds: {
+      directory: './seeds/dev'
+    },
+    useNullAsDefault: true
+  },
+
+  production: {
+    client: 'pg',
+    connection: 'process.env.DATABASE_URL'
```

Comment:
I think you have to add the +ssql to the end of the production connection so that the heroku server addon will work with it but it can be added later

Reply:
FIXED!!